test/integration/webhook_test.go

```diff
@@ -42,13 +42,13 @@ const highEndConsumerUsageCount = 50
 
 func TestValidationWebhook(t *testing.T) {
 	t.Parallel()
-	ns, cleanup := namespace(t)
-	defer cleanup()
-
 	if env.Cluster().Type() != kind.KindClusterType {
 		t.Skip("TODO: webhook tests are only supported on KIND based environments right now")
 	}
 
+	ns, cleanup := namespace(t)
+	defer cleanup()
+
```

Comment:
So technically speaking this change isn't quite right: the `namespace()` function doesn't actually _create_ the namespace, it retrieves one that will be created in _any_ case by the test suite. It doesn't really have any effect on the testing, but as such we should leave the cleanup where it was as it otherwise wont actually get cleaned up if the test is skipped.

Reply:
done